### PR TITLE
Add translations for AR, TR, HU, ID, IT, JA languages

### PR DESCRIPTION
**Pull request by esjs**
## What does this PR do?

New translations added

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

### CODE REVIEW

File: src/locales/hu/common.ftl [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+common-banEmailTemplate =
+  Helló { $username },
+
+  Valaki, aki hozzáfér a fiókodhoz megsértette a közösségi irányelveinket. Ennek eredményeként kitiltottuk a fiókodat. Ezentúl nem fogsz tudni hozzászólni, reagálni, hozzászólásokat jelenteni.
+
+common-embedNotFound = A kért tartalom nem elérhető. Lehet, hogy törölték.
+common-embedInvalid = The URL for this external media is invalid.
+common-networkError = Hálózati hiba. Kérjük, frissítse az oldalt, és próbálja meg újra.
+common-embedInternalError = Belső hibát észleltünk a média betöltése közben
+.common-embedTypeNotSupported = Nem támogatott média.
```
Comment (kabeaty) [inline src/locales/hu/common.ftl:13]: Can you please remove the period at the beginning of this translation key?

File: src/locales/it-IT/admin.ftl [hunk 2]
```diff
@@ -0,0 +1,1512 @@
...
+configure-moderation-akismet-ipBased = Rilevamento dello spam basato su IP
+configure-moderation-akismet-accountNote =
+  Nota: è necessario aggiungere il/i tuo/i dominio/i attivo/i
+Nel tuo account Akismet: <externalLink>https://akismet.com/account/</externalLink>
+configure-moderation-akismet-siteURL = URL sito
+
+
+#### Perspective
+configure-moderation-perspective-title = Filtro per i commenti tossici
+configure-moderation-perspective-explanation =
```
Comment (kabeaty) [inline src/locales/it-IT/admin.ftl]: It looks like the rest of this file is in English. Did you mean to include more Italian translations here?

File: src/locales/ja-JP/admin.ftl [hunk 3]
```diff
@@ -0,0 +1,1537 @@
...
+  サイト全体で新しいコメントを有効にする
+configure-general-sitewideCommenting-onCommentStreamsOpened =
+  オン-新しいコメントのために開かれたコメントストリーム
+configure-general-sitewideCommenting-offCommentStreamsClosed =
+  オフ-新しいコメントのためにコメントストリームを閉じました
+configure-general-sitewideCommenting-message = サイト全体のクローズドコメントメッセージ
+configure-general-sitewideCommenting-messageExplanation =
+  コメントストリームがサイト全体で閉じられたときに表示されるメッセージを記述します
+
+#### Embed Links
```
Comment (kabeaty) [inline src/locales/ja-JP/admin.ftl]: It looks like the rest of this file is in English. Did you mean to include more Japanese translations here?

File: src/locales/ja-JP/stream.ftl [hunk 4]
```diff
@@ -0,0 +1,895 @@
...
+comments-sortMenu-oldest = 最も古い
+comments-sortMenu-mostReplies = もっと返信を見る
+
+comments-userPopover =
+  .description = より多くのユーザー情報を含むポップオーバー
+comments-userPopover-memberSince = { DATETIME($timestamp, year: "numeric", month: "long", day: "numeric") } 以降のメンバー
+comments-userPopover-ignore = 無視する
+
+comments-userIgnorePopover-ignoreUser = {$username} を無視しますか？
+comments-userIgnorePopover-description =
```
Comment (kabeaty) [inline src/locales/ja-JP/stream.ftl]: It looks like this translation is English. Did you mean to include a Japanese translation here?

File: src/locales/ja-JP/stream.ftl [hunk 5]
```diff
@@ -0,0 +1,895 @@
...
+
+comments-jumpToComment-title = あなたの返信は、下記に投稿されています
+comments-jumpToComment-GoToReply = 返信する
+
+comments-mobileToolbar-closeButton =
+  .aria-label = 閉じる
+comments-mobileToolbar-unmarkAll = 全てのマークを外す
+comments-mobileToolbar-nextUnread = 次の未読コメント
+
+comments-replyChangedWarning-theCommentHasJust =
```
Comment (kabeaty) [inline src/locales/ja-JP/stream.ftl]: It looks like this translation is English. Did you mean to include a Japanese translation here?

**Review by kabeaty — COMMENTED** [review]
Thank you for adding these translations! There are a few files containing English where I left comments to confirm if you meant to include more Italian/Japanese translations in them instead.

**Comment by esjs** [comment]
> Thank you for adding these translations! There are a few files containing English where I left comments to confirm if you meant to include more Italian/Japanese translations in them instead.

Those translations were provided by editors of our website. I think it was hard for them to translate everything due to lack of context and ability to check the end result. We decided to submit translations to you as is and fine-tune them afterward if it's possible.

**Comment by kabeaty** [comment]
> > Thank you for adding these translations! There are a few files containing English where I left comments to confirm if you meant to include more Italian/Japanese translations in them instead.
> 
> Those translations were provided by editors of our website. I think it was hard for them to translate everything due to lack of context and ability to check the end result. We decided to submit translations to you as is and fine-tune them afterward if it's possible.

Okay, that makes sense, thanks for confirming! In this case, do you mind removing the English translations and their translation keys from the files for now? This way, it won't cause any conflicts if the English changes, and any translations missing now will still fall back to English anyway. 

After those updates are in, this looks good to me!

**Comment by esjs** [comment]
> Okay, that makes sense, thanks for confirming! In this case, do you mind removing the English translations and their translation keys from the files for now? This way, it won't cause any conflicts if the English changes, and any translations missing now will still fall back to English anyway.
> 
> After those updates are in, this looks good to me!

@kabeaty Requested changes are done, please consider reviewing it once again.

**Review by kabeaty — APPROVED** [review]
Looks good to me! Thank you again for making the updates and adding the translations.

**Pull request merged**